api/proto/message/message.proto

```diff
@@ -86,6 +86,7 @@ message ConsensusRequest {
   bytes block_hash = 4;
   bytes sender_pubkey = 5;
   bytes payload = 6;
+  bytes block = 7;
```

Comment:
Move block under block_hash with identifier 5 "block = 5;".   we can still refresh the protobuf definition now. Let's do it.